stable/yourls/templates/deployment.yaml

```
@@ -0,0 +1,173 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "yourls.fullname" . }}
+  labels:
+    app: "{{ template "yourls.fullname" . }}"
+    chart: "{{ template "yourls.chart" . }}"
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
+spec:
+  selector:
+    matchLabels:
+      app: "{{ template "yourls.fullname" . }}"
+      release: {{ .Release.Name | quote }}
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: "{{ template "yourls.fullname" . }}"
+        chart: "{{ template "yourls.chart" . }}"
+        release: {{ .Release.Name | quote }}
+      {{- if or .Values.podAnnotations .Values.metrics.enabled }}
+      annotations:
+      {{- if .Values.podAnnotations }}
+{{ toYaml .Values.podAnnotations | indent 8 }}
+      {{- end }}
+      {{- if .Values.metrics.podAnnotations }}
+{{ toYaml .Values.metrics.podAnnotations | indent 8 }}
+      {{- end }}
+      {{- end }}
+    spec:
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.image.pullSecrets }}
+        - name: {{ . }}
+      {{- end}}
+      {{- end }}
+      hostAliases:
+        - ip: "127.0.0.1"
+          hostnames:
+            - "status.localhost"
+      containers:
+        - name: "{{ template "yourls.fullname" . }}"
+          image: {{ template "yourls.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          env:
+            - name: ALLOW_EMPTY_PASSWORD
+            {{- if .Values.allowEmptyPassword }}
+              value: "yes"
+            {{- else }}
+              value: "no"
+            {{- end }}
+            - name: YOURLS_DB_HOST
+            {{- if .Values.mariadb.enabled }}
+              value: {{ template "mariadb.fullname" . }}:3306
```

```suggestion
              value: {{ template "mariadb.fullname" . }}:{{ .Values.mariadb.service.port }}
```
(Needs to be added to the `values.yaml` file, I'd stay with the default `3306` value).